pyproject.toml

```diff
@@ -59,6 +59,9 @@ extra-dependencies = [
   "fastapi",
   # Tool
   "jsonschema",
+  # LLMMetadataExtractor dependencies
+  "amazon-bedrock-haystack==1.0.2",
+  "google-vertex-haystack==2.0.0",
```

Comment:
Maybe we should make these `>=` to be consistent with how we are handling the other deps here? It seems like aren't fixing the versions of other haystack integration libraries.